src/databases/Blueprint/avtBlueprintFileFormat.C

```diff
@@ -747,6 +744,88 @@ avtBlueprintFileFormat::AddBlueprintMeshAndFieldMetadata(avtDatabaseMetaData *md
     }
 }
 
+// ****************************************************************************
```

Comment:
I refactored expression logic to own function to facilitate filtering expression variable names which may include prepending mesh names already (think CMFEs). I needed a way to determine of a given variable name in an expression already had mesh name prepended.